infer/lib/rust/tests/programs/operands/move/struct_move.sil

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright (c) Facebook, Inc. and its affiliates. 
+//
+// This source code is licensed under the MIT license found in the 
+// LICENSE file in the root directory of this source tree. 
+
+.source_language = "rust"
+
+define test() : void {
+  local var_0: void, var_1: Point, var_2: Point
+
+  #node_0:
+    %0 = Point { x: 1, y: 2 }
+    store &var_1 <- %0
```

Comment:
You need to add the type decleration, I will work on updating the rules for structs more.
```
type Point = {x : int; y : int}
```
```suggestion
    store &var_1.Point.x <- 1
    store &var_1.Point.y <- 2
```